bluemira/utilities/opt_variables.py

```diff
@@ -265,117 +279,167 @@ def __str__(self) -> str:
 
         return f"{self.name} = {self.value}{bound}{descr}"
 
+    def __add__(self, other: "OptVariable"):
```

Comment:
add `from __future__ import annotations` to the top of the imports list and then:
```suggestion
    def __add__(self, other: OptVariable):
```